File: Fibonacci/Fibonacci.c
#include <stdio.h>
#include <stdlib.h>

int Fibonacci(int n) {
    int result;
    if (n < 0) {
        fprintf(stderr, "%d is less than 0", n);
        exit(1);
    } else if (n <= 1) {
        result = n;
    } else {
        result = Fibonacci(n - 1) + Fibonacci(n - 2);
    }
    return result;
}

int main(void) {
    for (int i = 0; i < 10; i++) {
        printf("Fib of %d: %d\n", i, Fibonacci(i));
    }
    // Fibonacci(-1);
    return 0;
}

File: Fibonacci/Fibonacci.py
class LessThanZeroException(Exception):
    pass

def Fibonacci(n):
    if n < 0:
        raise LessThanZeroException("{} is less than 0".format(n))
    elif n <= 1:
        result = n
    else:
        result = Fibonacci(n - 1) + Fibonacci(n - 2)
    return result

if __name__ == "__main__":
    for i in range(0, 10):
        print("Fib of {}: {}".format(i, Fibonacci(i)))
    #Fibonacci(-1);